core/templates/pages/donate-page/donate-page.component.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the donate page.
+ */
+
+import { Component, OnInit } from '@angular/core';
+
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
+import { SiteAnalyticsService } from 'services/site-analytics.service.ts';
+import { WindowDimensionsService } from
+  'services/contextual/window-dimensions.service.ts';
+
+require('base-components/base-content.directive.ts');
+
+@Component({
+  selector: 'donate-page',
+  template: require('./donate-page.component.html'),
+  styleUrls: []
```

Comment:
@bansalnitish I was also thinking maybe we could remove the style element from HTML and add those styles to a file called `donate-page.component.css`.

@nishantwrp Similar question here again. What changes would we have to make in the web-pack config to just use URLs. Ideally it should look like:
```
@Component({
  selector: 'donate-page',
  templateUrl:  './donate-page.component.html',
  styleUrls: ['./donate-page.component.css']
})
```

Reply:
Not removing css out of html right now. The code refactoring can be done at some later stage since it requires some webpack configurations which might take time.